vaadin-time-picker-flow-parent/vaadin-time-picker-flow/src/main/java/com/vaadin/flow/component/timepicker/TimePicker.java

```diff
@@ -34,6 +34,7 @@
 import com.vaadin.flow.component.HasValue;
 import com.vaadin.flow.component.UI;
 import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.component.HasClearButton;
```

Comment:
nit: let's place this import before the `com.vaadin.flow.component.HasEnabled` import.

Reply:
Done